src/Pages/Auth/Register.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import logo from "../../assets/budget-it.png";
+import { currency } from "./currency";
+const Register = () => {
+  const [user, setUser] = useState({
+    firstname: "",
+    lastname: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+    currency: "NGN",
+  });
+
+  const handleChange = (e) => {
+    setUser({
+      ...user,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const handleRegSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className="auth">
+      <div className="side">
+        <img src={logo} alt="BudgetIt logo" />
+        <h3> Enter your details</h3>
+        <span> Budget your spending.</span>
+        <span>Budgeting never felt so good.</span>
+      </div>
+      <div className="reg-form">
+        <form onSubmit={handleRegSubmit}>
+          <label className="name">First name</label>
```

Comment:
Wrap each of the input/label in a div. But the the first and last name input, wrap them in another div and use flex box to put the space between them. Using absolute positioning won't be good for some screens

Reply:
Alright